bin/bundix

```diff
@@ -1,92 +1,97 @@
-#!/usr/bin/env ruby
+#! /usr/bin/env ruby
 
 require 'optparse'
 require 'tmpdir'
 require 'pathname'
 
-require_relative '../lib/bundix'
-
-options = {
-  ruby: 'ruby',
-  bundle_pack_path: 'vendor/bundle',
-  lockfile: 'Gemfile.lock',
-  gemset: 'gemset.nix'
-}
-
-op = OptionParser.new do |o|
-  o.on '-m', '--magic', 'lock, pack, and write dependencies' do
-    options[:magic] = true
-  end
-
-  o.on '--ruby=ruby', 'ruby version to use for magic and init, defaults to latest' do |value|
-    options[:ruby] = value
-  end
-
-  o.on '--bundle-pack-path=vendor/bundle', "path to pack the magic" do |value|
-    options[:bundle_pack_path] = value
-  end
-
-  o.on '-i', '--init', "initialize a new shell.nix for nix-shell (won't overwrite old ones)" do
-    options[:init] = true
-  end
-
-  o.on '--gemset=gemset.nix', 'path to the gemset.nix' do |value|
-    options[:gemset] = File.expand_path(value)
-  end
+ENV.delete('SSL_CERT_FILE') if ENV['SSL_CERT_FILE'] == '/no-cert-file.crt'
```

Comment:
I'm using it like `nix-shell -p bundler --command ~/github/manveru/bundix/bin/bundix`, and having no SSL connections means bundix can't work. If you know a more elegant solution, please inform me :)